app/templates/complete_display_image.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<a href="/Return">Return to home page</a>
```

Comment:
this file should be in same style as other pages like signup, login.html. we should utilize flask's build-in properties. please refer to lastest master branch for the link management and proper format of the html files.